search-resources-dynamodb-trigger/src/test/java/no/unit/nva/publication/DynamoDBStreamHandlerTest.java

```diff
@@ -369,30 +369,7 @@ private DeleteResponse mockDeleteResponse() {
     }
 
 
-    //
-    //
```

Comment:
Test was deleted, because we do not do anything specific with incomplete dates. Right now we do as before, namely we copy the year month and date. This is tested in previous tests.